src/print.c

```diff
@@ -42,6 +43,15 @@
 
 #include "gtkbackports.h"
 
+#define DPI 150.0
```

Comment:
Document origin of this magic number?

Reply:
Fixed by 7ec9062903cce6 and 3a6cd1e

Reply:
I'm still not sure I understand the reasoning behind "my choice".

Reply:
I chose 150 as the print resolution, I hesitated with 300 but that corresponds more to a print resolution for graphic designer.

Reply:
Please edit this comment to say that. Something along the lines of "150 is a reasonable resolution for consumer printing".

Reply:
Fixed